js/components/discussions/threads-form.vue

```diff
@@ -36,15 +35,13 @@ export default {
          * Prefill the form and focus the comment area
          */
         prefill(title, comment) {
+            comment = comment || '';
             this.title = title || '';
-            this.comment = comment || '';
+            this.comment = comment;
```

Comment:
it was written like that before but the comment is reused below twice and it needs to be a string everywhere so I factorised.